ddtrace/utils/wrappers.py

```diff
@@ -0,0 +1,65 @@
+import wrapt
+import inspect
+
+from .deprecation import deprecated
+
+
+def unwrap(obj, attr):
+    f = getattr(obj, attr, None)
+    if f and isinstance(f, wrapt.ObjectProxy) and hasattr(f, '__wrapped__'):
+        setattr(obj, attr, f.__wrapped__)
+
+
+@deprecated('`wrapt` library is used instead', version='1.0.0')
```

Comment:
This function is never used in our repo and it should not be used by anyone. But since there is no reason to remove it in this minor release, I would prefer to keep it till 1.0.0 where we can definitely clean a lot our code.